### PR TITLE
Cogehr/default days filter

### LINKED ISSUE #3: Ensure queries don't return data older than fourteen days
This needs to be included in all query calls, _except_ `GetAsync(string)` because that call by definition should always retrieve the `MatchMessageRecord` which matches the provided ID.

I use fourteen in the title because that's the current requirement, but ideally this would be configurable in the App Service/appsettings.json file in case this would ever change (and for future-proofing).

### PR DESCRIPTION
**Pull request by corygehr**
Resolves #3.

### CODE REVIEW

File: CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs [hunk 1]
```diff
@@ -43,6 +42,17 @@ public CosmosMatchMessageRepository(CosmosContext dbContext) : base(dbContext)
             );
         }
 
+        /// <summary>
+        /// Returns the oldest allowed data timestamp for a Cosmos record
+        /// </summary>
+        /// <returns>Oldest allowed timestamp, in ms since UNIX epoch</returns>
+        private long _getMaxDataAge()
```
Comment (ataymano) [inline CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs]: [nit] "Age" seems like not absolute timestamp but (Now() - OldestTimestamp()) (which is 14 days in our case), maybe something like getOldestAllowedTimestamp

File: CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs [hunk 2]
```diff
@@ -86,6 +96,7 @@ public async Task<IEnumerable<MessageInfo>> GetLatestAsync(Region region, long l
             var iterator = queryable
                 .Where(r =>
                     r.Timestamp > lastTimestamp
+                    && r.Timestamp >= this._getMaxDataAge()
```
Comment (ataymano) [inline CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs]: [nit] maybe something like timestampThreshold = Max(lastTimestamp, getMaxDataAge()) and single condition in query is abit cleaner?

Comment (corygehr) [reply CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs]: I like this idea much better.

File: CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs (hunk 1 above)
Comment (corygehr) [reply CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs]: Similar to your below, I think that makes more sense. I'll leave the configuration parameter name unchanged but will update this function name.

**Pull request merged**